build-tools/linux-systemd-service/mintlayer-wallet-rpc-testnet.service

```diff
@@ -0,0 +1,43 @@
+# For mainnet, change every testnet instance to mainnet
+[Unit]
+Description=Mintlayer Testnet Wallet RPC Daemon
+After=syslog.target network-online.target
+
+# RULES for picking a user:
+#  1. DO NOT use root
+#  2. DO NOT use a user that has root access through sudo, doas, or similar
+[Service]
+User=mintlayeruser
+
+# Type is simple because the wallet-rpc-daemon is blocking
+Type=simple
+
+# Path of the source code OR executable
+WorkingDirectory=/home/mintlayeruser/mintlayer-core
+
+# Use this if you want the wallet-rpc-daemon to be built from source
+# Consider running the `wallet-rpc-daemon` with the `--help` option to see what options are available (authentication, etc)
+# --start-staking-for-account 0 will start staking for account number 0
+ExecStart=cargo run -j 1 --release --bin wallet-rpc-daemon -- testnet /path/to/wallet/file --start-staking-for-account 0
+# If you're using an executable and didn't clone the source, comment the line above, and uncomment below
```

Comment:
It might be worth mentioning that -j specifies the number of cores to use when building and that it can be set to a higher value to speed it up (provided that extra cores are available and the machine has enough RAM).

Reply:
It's done like that on purpose. This is a service, so shouldn't consume so many resources when running. I don't think it's worth mentioning.